packages/playground/src/pages/Cards/Cards.js

```diff
@@ -5,16 +5,15 @@ import { handleSubmit, handleAdditionalDetails, handleError } from '../../handle
 import { amount, shopperLocale } from '../../config/commonConfig';
 import '../../../config/polyfills';
 import '../../style.scss';
-import { countryCode } from '../../config/commonConfig';
 
 getPaymentMethods({ amount, shopperLocale }).then(paymentMethodsResponse => {
     window.checkout = new AdyenCheckout({
         amount,
         clientKey: process.env.__CLIENT_KEY__,
         paymentMethodsResponse,
         locale: shopperLocale,
-        countryCode,
```

Comment:
Is this deleted line intentional?

Reply:
yes. We only added it to make the kcp tests work. Now we have the clientScripts we don't need it anymore